panic: runtime error: invalid memory address or nil pointer dereference

First thanks for the awesome code!! this is such a great project. I am having a little trouble here wondering if anyone have seen this before

```
$ terraform version
Terraform v0.12.29
+ provider.zabbix (unversioned)
$ go version
go version go1.14.1 darwin/amd64

```


I am in linux_general folder trying to apply as follow
```
$  terraform plan -var-file="terraform.tfvars" 
```

full crash log here
```
$ cat crash.log 
2020/12/01 19:49:03 [INFO] Terraform version: 0.12.29  
2020/12/01 19:49:03 [INFO] Go runtime version: go1.12.13
2020/12/01 19:49:03 [INFO] CLI args: []string{"/Users/yen/terraform/terraform", "plan", "-var-file=terraform.tfvars"}
2020/12/01 19:49:03 [DEBUG] Attempting to open CLI config file: /Users/yen/.terraformrc
2020/12/01 19:49:03 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2020/12/01 19:49:03 [DEBUG] checking for credentials in "/Users/yen/.terraform.d/plugins"
2020/12/01 19:49:03 [DEBUG] checking for credentials in "/Users/yen/.terraform.d/plugins/darwin_amd64"
2020/12/01 19:49:03 [INFO] CLI command args: []string{"plan", "-var-file=terraform.tfvars"}
2020/12/01 19:49:03 [TRACE] Meta.Backend: no config given or present on disk, so returning nil config
2020/12/01 19:49:03 [TRACE] Meta.Backend: backend has not previously been initialized in this working directory
2020/12/01 19:49:03 [DEBUG] New state was assigned lineage "046cf3d5-9fb5-10fc-9281-8b2cc9511e35"
2020/12/01 19:49:03 [TRACE] Meta.Backend: using default local state only (no backend configuration, and no existing initialized backend)
2020/12/01 19:49:03 [TRACE] Meta.Backend: instantiated backend of type <nil>
2020/12/01 19:49:03 [DEBUG] checking for provider in "."
2020/12/01 19:49:03 [DEBUG] checking for provider in "/Users/yen/terraform"
2020/12/01 19:49:03 [DEBUG] checking for provider in ".terraform/plugins/darwin_amd64"
2020/12/01 19:49:03 [DEBUG] checking for provider in "/Users/yen/.terraform.d/plugins"
2020/12/01 19:49:03 [DEBUG] found provider "terraform-provider-ibm_v0.17.0"
2020/12/01 19:49:03 [DEBUG] found provider "terraform-provider-ibm_v0.17.1.old"
2020/12/01 19:49:03 [DEBUG] checking for provider in "/Users/yen/.terraform.d/plugins/darwin_amd64"
2020/12/01 19:49:03 [WARN] found legacy provider "terraform-provider-zabbix"
2020/12/01 19:49:03 [DEBUG] found valid plugin: "zabbix", "0.0.0", "/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix"
2020/12/01 19:49:03 [DEBUG] found valid plugin: "ibm", "0.17.0", "/Users/yen/.terraform.d/plugins/terraform-provider-ibm_v0.17.0"
2020/12/01 19:49:03 [DEBUG] checking for provisioner in "."
2020/12/01 19:49:03 [DEBUG] checking for provisioner in "/Users/yen/terraform"
2020/12/01 19:49:03 [DEBUG] checking for provisioner in ".terraform/plugins/darwin_amd64"
2020/12/01 19:49:03 [DEBUG] checking for provisioner in "/Users/yen/.terraform.d/plugins"
2020/12/01 19:49:03 [DEBUG] checking for provisioner in "/Users/yen/.terraform.d/plugins/darwin_amd64"
2020/12/01 19:49:03 [TRACE] Meta.Backend: backend <nil> does not support operations, so wrapping it in a local backend
2020/12/01 19:49:03 [INFO] backend/local: starting Plan operation
2020/12/01 19:49:03 [TRACE] backend/local: requesting state manager for workspace "default"
2020/12/01 19:49:03 [TRACE] backend/local: state manager for workspace "default" will:
 - read initial snapshot from terraform.tfstate
 - write new snapshots to terraform.tfstate
 - create any backup at terraform.tfstate.backup
2020/12/01 19:49:03 [TRACE] backend/local: requesting state lock for workspace "default"
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: preparing to manage state snapshots at terraform.tfstate
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: existing snapshot has lineage "c6156896-b493-7ff1-9756-8b5e9b391a5e" serial 1
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: locking terraform.tfstate using fcntl flock
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: writing lock metadata to .terraform.tfstate.lock.info
2020/12/01 19:49:03 [TRACE] backend/local: reading remote state for workspace "default"
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: reading latest snapshot from terraform.tfstate
2020/12/01 19:49:03 [TRACE] statemgr.Filesystem: read snapshot with lineage "c6156896-b493-7ff1-9756-8b5e9b391a5e" serial 1
2020/12/01 19:49:03 [TRACE] backend/local: retrieving local state snapshot for workspace "default"
2020/12/01 19:49:03 [TRACE] backend/local: building context for current working directory
2020/12/01 19:49:03 [TRACE] terraform.NewContext: starting
2020/12/01 19:49:03 [TRACE] terraform.NewContext: resolving provider version selections
2020/12/01 19:49:04 [TRACE] terraform.NewContext: loading provider schemas
2020/12/01 19:49:04 [TRACE] LoadSchemas: retrieving schema for provider type "zabbix"
2020-12-01T19:49:04.067-0600 [INFO]  plugin: configuring client automatic mTLS
2020-12-01T19:49:04.095-0600 [DEBUG] plugin: starting plugin: path=/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix args=[/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix]
2020-12-01T19:49:04.098-0600 [DEBUG] plugin: plugin started: path=/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix pid=15543
2020-12-01T19:49:04.098-0600 [DEBUG] plugin: waiting for RPC address: path=/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix
2020-12-01T19:49:04.115-0600 [INFO]  plugin.terraform-provider-zabbix: configuring server automatic mTLS: timestamp=2020-12-01T19:49:04.115-0600
2020-12-01T19:49:04.143-0600 [DEBUG] plugin: using plugin: version=5
2020-12-01T19:49:04.143-0600 [DEBUG] plugin.terraform-provider-zabbix: plugin address: network=unix address=/var/folders/42/bg6787l131n4rx6klkk823v40000gn/T/plugin439298554 timestamp=2020-12-01T19:49:04.143-0600
2020/12/01 19:49:04 [TRACE] GRPCProvider: GetSchema
2020-12-01T19:49:04.195-0600 [TRACE] plugin.stdio: waiting for stdio data
2020-12-01T19:49:04.195-0600 [WARN]  plugin.stdio: received EOF, stopping recv loop: err="rpc error: code = Unimplemented desc = unknown service plugin.GRPCStdio"
2020/12/01 19:49:04 [TRACE] GRPCProvider: Close
2020-12-01T19:49:04.198-0600 [DEBUG] plugin: plugin process exited: path=/Users/yen/.terraform.d/plugins/darwin_amd64/terraform-provider-zabbix pid=15543
2020-12-01T19:49:04.198-0600 [DEBUG] plugin: plugin exited
2020/12/01 19:49:04 [TRACE] terraform.NewContext: complete
2020/12/01 19:49:04 [TRACE] backend/local: finished building terraform.Context
2020/12/01 19:49:04 [TRACE] backend/local: requesting interactive input, if necessary
2020/12/01 19:49:04 [TRACE] Context.Input: Prompting for provider arguments
2020/12/01 19:49:04 [TRACE] Context.Input: Provider provider.zabbix declared at providers.tf:1,1-18
2020/12/01 19:49:04 [TRACE] Context.Input: Input for provider.zabbix: map[string]cty.Value{}
2020/12/01 19:49:04 [TRACE] backend/local: running validation operation
2020/12/01 19:49:04 [INFO] terraform: building graph: GraphTypeValidate
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.ConfigTransformer
2020/12/01 19:49:04 [TRACE] ConfigTransformer: Starting for path: 
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.ConfigTransformer with new graph:
  zabbix_host_group.template_linux - *terraform.NodeValidatableResource
  zabbix_item.cpu_load_avg1 - *terraform.NodeValidatableResource
  zabbix_item.cpu_num_online - *terraform.NodeValidatableResource
  zabbix_item.cpu_util_idle - *terraform.NodeValidatableResource
  zabbix_item.memory_size_pavailable - *terraform.NodeValidatableResource
  zabbix_item.ssh_server_is_running - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_inodes - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_space - *terraform.NodeValidatableResource
  zabbix_lld_rule.filesystem_discovery_rule - *terraform.NodeValidatableResource
  zabbix_lld_rule_link.test_link - *terraform.NodeValidatableResource
  zabbix_template.base_linux_filesystem - *terraform.NodeValidatableResource
  zabbix_template.base_linux_general - *terraform.NodeValidatableResource
  zabbix_template.base_linux_network - *terraform.NodeValidatableResource
  zabbix_template.standard_linux - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_filesystem_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_general_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_network_link - *terraform.NodeValidatableResource
  zabbix_template_link.standard_linux_link - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_warn - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_warn - *terraform.NodeValidatableResource
  ------
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.LocalTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.LocalTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.OutputTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.OutputTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.OrphanResourceInstanceTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.OrphanResourceInstanceTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.StateTransformer
2020/12/01 19:49:04 [TRACE] StateTransformer: creating nodes for deposed instance objects only
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.StateTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.OrphanOutputTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.OrphanOutputTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.AttachResourceConfigTransformer
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_lld_rule.filesystem_discovery_rule" (*terraform.NodeValidatableResource) config from lld_rule.tf:1,1-55
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_inodes_disaster" (*terraform.NodeValidatableResource) config from lld_rule.tf:39,1-64
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item.memory_size_pavailable" (*terraform.NodeValidatableResource) config from items.tf:28,1-48
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item.cpu_load_avg1" (*terraform.NodeValidatableResource) config from items.tf:1,1-39
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_space_disaster" (*terraform.NodeValidatableResource) config from lld_rule.tf:72,1-63
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_inodes_high" (*terraform.NodeValidatableResource) config from lld_rule.tf:45,1-60
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_inodes_warn" (*terraform.NodeValidatableResource) config from lld_rule.tf:63,1-60
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item_prototype.free_disk_inodes" (*terraform.NodeValidatableResource) config from lld_rule.tf:17,1-52
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_space_high" (*terraform.NodeValidatableResource) config from lld_rule.tf:78,1-59
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template_link.standard_linux_link" (*terraform.NodeValidatableResource) config from templates.tf:107,1-54
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template_link.base_linux_general_link" (*terraform.NodeValidatableResource) config from templates.tf:24,1-58
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template.base_linux_network" (*terraform.NodeValidatableResource) config from templates.tf:62,1-48
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template_link.base_linux_filesystem_link" (*terraform.NodeValidatableResource) config from templates.tf:83,1-61
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_space_warn" (*terraform.NodeValidatableResource) config from lld_rule.tf:96,1-59
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_lld_rule_link.test_link" (*terraform.NodeValidatableResource) config from lld_rule.tf:105,1-44
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item.ssh_server_is_running" (*terraform.NodeValidatableResource) config from items.tf:37,1-47
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_space_avg" (*terraform.NodeValidatableResource) config from lld_rule.tf:87,1-58
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template.standard_linux" (*terraform.NodeValidatableResource) config from templates.tf:91,1-44
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item.cpu_num_online" (*terraform.NodeValidatableResource) config from items.tf:19,1-40
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_trigger_prototype.free_disk_inodes_avg" (*terraform.NodeValidatableResource) config from lld_rule.tf:54,1-59
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template_link.base_linux_network_link" (*terraform.NodeValidatableResource) config from templates.tf:68,1-58
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item_prototype.free_disk_space" (*terraform.NodeValidatableResource) config from lld_rule.tf:28,1-51
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_host_group.template_linux" (*terraform.NodeValidatableResource) config from host_groups.tf:1,1-46
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template.base_linux_filesystem" (*terraform.NodeValidatableResource) config from templates.tf:78,1-51
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_item.cpu_util_idle" (*terraform.NodeValidatableResource) config from items.tf:10,1-39
2020/12/01 19:49:04 [TRACE] AttachResourceConfigTransformer: attaching to "zabbix_template.base_linux_general" (*terraform.NodeValidatableResource) config from templates.tf:1,1-48
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.AttachResourceConfigTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.AttachStateTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.AttachStateTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.RootVariableTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.RootVariableTransformer with new graph:
  var.password - *terraform.NodeRootVariable
  var.server_url - *terraform.NodeRootVariable
  var.user - *terraform.NodeRootVariable
  zabbix_host_group.template_linux - *terraform.NodeValidatableResource
  zabbix_item.cpu_load_avg1 - *terraform.NodeValidatableResource
  zabbix_item.cpu_num_online - *terraform.NodeValidatableResource
  zabbix_item.cpu_util_idle - *terraform.NodeValidatableResource
  zabbix_item.memory_size_pavailable - *terraform.NodeValidatableResource
  zabbix_item.ssh_server_is_running - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_inodes - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_space - *terraform.NodeValidatableResource
  zabbix_lld_rule.filesystem_discovery_rule - *terraform.NodeValidatableResource
  zabbix_lld_rule_link.test_link - *terraform.NodeValidatableResource
  zabbix_template.base_linux_filesystem - *terraform.NodeValidatableResource
  zabbix_template.base_linux_general - *terraform.NodeValidatableResource
  zabbix_template.base_linux_network - *terraform.NodeValidatableResource
  zabbix_template.standard_linux - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_filesystem_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_general_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_network_link - *terraform.NodeValidatableResource
  zabbix_template_link.standard_linux_link - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_warn - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_warn - *terraform.NodeValidatableResource
  ------
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.MissingProvisionerTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.MissingProvisionerTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.ProvisionerTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.ProvisionerTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.ModuleVariableTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.ModuleVariableTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.graphTransformerMulti
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Executing graph transform *terraform.ProviderConfigTransformer
2020/12/01 19:49:04 [TRACE] ProviderConfigTransformer: attaching to "provider.zabbix" provider configuration from providers.tf:1,1-18
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Completed graph transform *terraform.ProviderConfigTransformer with new graph:
  provider.zabbix - *terraform.NodeApplyableProvider
  var.password - *terraform.NodeRootVariable
  var.server_url - *terraform.NodeRootVariable
  var.user - *terraform.NodeRootVariable
  zabbix_host_group.template_linux - *terraform.NodeValidatableResource
  zabbix_item.cpu_load_avg1 - *terraform.NodeValidatableResource
  zabbix_item.cpu_num_online - *terraform.NodeValidatableResource
  zabbix_item.cpu_util_idle - *terraform.NodeValidatableResource
  zabbix_item.memory_size_pavailable - *terraform.NodeValidatableResource
  zabbix_item.ssh_server_is_running - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_inodes - *terraform.NodeValidatableResource
  zabbix_item_prototype.free_disk_space - *terraform.NodeValidatableResource
  zabbix_lld_rule.filesystem_discovery_rule - *terraform.NodeValidatableResource
  zabbix_lld_rule_link.test_link - *terraform.NodeValidatableResource
  zabbix_template.base_linux_filesystem - *terraform.NodeValidatableResource
  zabbix_template.base_linux_general - *terraform.NodeValidatableResource
  zabbix_template.base_linux_network - *terraform.NodeValidatableResource
  zabbix_template.standard_linux - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_filesystem_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_general_link - *terraform.NodeValidatableResource
  zabbix_template_link.base_linux_network_link - *terraform.NodeValidatableResource
  zabbix_template_link.standard_linux_link - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_inodes_warn - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_avg - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_disaster - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_high - *terraform.NodeValidatableResource
  zabbix_trigger_prototype.free_disk_space_warn - *terraform.NodeValidatableResource
  ------
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Executing graph transform *terraform.MissingProviderTransformer
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Completed graph transform *terraform.MissingProviderTransformer (no changes)
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Executing graph transform *terraform.ProviderTransformer
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_inodes_warn is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item_prototype.free_disk_inodes is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_lld_rule.filesystem_discovery_rule is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_inodes_disaster is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item.memory_size_pavailable is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_space_disaster is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_space_high is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template.base_linux_network is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item_prototype.free_disk_space is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item.cpu_util_idle is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_inodes_high is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item.cpu_load_avg1 is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_space_warn is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_lld_rule_link.test_link is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item.ssh_server_is_running is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template_link.standard_linux_link is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template_link.base_linux_general_link is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template_link.base_linux_filesystem_link is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template_link.base_linux_network_link is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_host_group.template_linux is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_space_avg is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template.standard_linux is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_item.cpu_num_online is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_trigger_prototype.free_disk_inodes_avg is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template.base_linux_filesystem is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: zabbix_template.base_linux_general is provided by provider.zabbix or inherited equivalent
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item_prototype.free_disk_inodes
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item_prototype.free_disk_inodes" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item_prototype.free_disk_space
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item_prototype.free_disk_space" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item.cpu_load_avg1
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item.cpu_load_avg1" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_lld_rule_link.test_link
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_lld_rule_link.test_link" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template_link.base_linux_filesystem_link
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template_link.base_linux_filesystem_link" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template_link.base_linux_network_link
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template_link.base_linux_network_link" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_inodes_avg
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_inodes_avg" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_inodes_warn
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_inodes_warn" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_inodes_disaster
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_inodes_disaster" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_space_high
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_space_high" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template.base_linux_network
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template.base_linux_network" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item.ssh_server_is_running
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item.ssh_server_is_running" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_host_group.template_linux
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_host_group.template_linux" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template.base_linux_general
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template.base_linux_general" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item.memory_size_pavailable
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item.memory_size_pavailable" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_space_disaster
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_space_disaster" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_space_warn
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_space_warn" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template_link.base_linux_general_link
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template_link.base_linux_general_link" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_space_avg
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_space_avg" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item.cpu_num_online
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item.cpu_num_online" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_lld_rule.filesystem_discovery_rule
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_lld_rule.filesystem_discovery_rule" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_item.cpu_util_idle
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_item.cpu_util_idle" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_trigger_prototype.free_disk_inodes_high
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_trigger_prototype.free_disk_inodes_high" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template_link.standard_linux_link
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template_link.standard_linux_link" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template.standard_linux
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template.standard_linux" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] ProviderTransformer: exact match for provider.zabbix serving zabbix_template.base_linux_filesystem
2020/12/01 19:49:04 [DEBUG] ProviderTransformer: "zabbix_template.base_linux_filesystem" (*terraform.NodeValidatableResource) needs provider.zabbix
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Completed graph transform *terraform.ProviderTransformer with new graph:
  provider.zabbix - *terraform.NodeApplyableProvider
  var.password - *terraform.NodeRootVariable
  var.server_url - *terraform.NodeRootVariable
  var.user - *terraform.NodeRootVariable
  zabbix_host_group.template_linux - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_load_avg1 - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_num_online - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_util_idle - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.memory_size_pavailable - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.ssh_server_is_running - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item_prototype.free_disk_inodes - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item_prototype.free_disk_space - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_lld_rule.filesystem_discovery_rule - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_lld_rule_link.test_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_filesystem - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_general - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_network - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.standard_linux - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_filesystem_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_general_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_network_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.standard_linux_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_avg - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_disaster - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_high - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_warn - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_avg - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_disaster - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_high - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_warn - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  ------
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Executing graph transform *terraform.PruneProviderTransformer
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Completed graph transform *terraform.PruneProviderTransformer (no changes)
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Executing graph transform *terraform.ParentProviderTransformer
2020/12/01 19:49:04 [TRACE] (graphTransformerMulti) Completed graph transform *terraform.ParentProviderTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.graphTransformerMulti with new graph:
  provider.zabbix - *terraform.NodeApplyableProvider
  var.password - *terraform.NodeRootVariable
  var.server_url - *terraform.NodeRootVariable
  var.user - *terraform.NodeRootVariable
  zabbix_host_group.template_linux - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_load_avg1 - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_num_online - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.cpu_util_idle - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.memory_size_pavailable - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item.ssh_server_is_running - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item_prototype.free_disk_inodes - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_item_prototype.free_disk_space - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_lld_rule.filesystem_discovery_rule - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_lld_rule_link.test_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_filesystem - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_general - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.base_linux_network - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template.standard_linux - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_filesystem_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_general_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.base_linux_network_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_template_link.standard_linux_link - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_avg - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_disaster - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_high - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_inodes_warn - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_avg - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_disaster - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_high - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  zabbix_trigger_prototype.free_disk_space_warn - *terraform.NodeValidatableResource
    provider.zabbix - *terraform.NodeApplyableProvider
  ------
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.RemovedModuleTransformer
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.RemovedModuleTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.AttachSchemaTransformer
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_inodes_warn
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item_prototype.free_disk_inodes
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_lld_rule.filesystem_discovery_rule
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_inodes_disaster
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item.memory_size_pavailable
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_space_disaster
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_space_high
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template.base_linux_network
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching provider config schema to provider.zabbix
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item_prototype.free_disk_space
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item.cpu_util_idle
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_inodes_high
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item.cpu_load_avg1
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_space_warn
2020/12/01 19:49:04 [ERROR] AttachSchemaTransformer: No resource schema available for zabbix_lld_rule_link.test_link
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item.ssh_server_is_running
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template_link.standard_linux_link
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template_link.base_linux_general_link
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template_link.base_linux_filesystem_link
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template_link.base_linux_network_link
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_host_group.template_linux
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_space_avg
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template.standard_linux
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_item.cpu_num_online
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_trigger_prototype.free_disk_inodes_avg
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template.base_linux_filesystem
2020/12/01 19:49:04 [TRACE] AttachSchemaTransformer: attaching resource schema to zabbix_template.base_linux_general
2020/12/01 19:49:04 [TRACE] Completed graph transform *terraform.AttachSchemaTransformer (no changes)
2020/12/01 19:49:04 [TRACE] Executing graph transform *terraform.ReferenceTransformer
2020/12/01 19:49:04 [WARN] no schema is attached to zabbix_lld_rule_link.test_link, so config references cannot be detected
2020/12/01 19:49:04 [TRACE] statemgr.Filesystem: removing lock metadata file .terraform.tfstate.lock.info
2020/12/01 19:49:04 [TRACE] statemgr.Filesystem: unlocking terraform.tfstate using fcntl flock
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1a0242e]

goroutine 15 [running]:
github.com/hashicorp/terraform/lang/blocktoattr.walkVariables(0x3067c40, 0xc00056d290, 0x0, 0xc0000ca001, 0x3067c40, 0xc00056d290, 0x0, 0x0, 0x0, 0x67)
	/home/circleci/project/project/lang/blocktoattr/variables.go:34 +0x1ae
github.com/hashicorp/terraform/lang/blocktoattr.ExpandedVariables(...)
	/home/circleci/project/project/lang/blocktoattr/variables.go:24
github.com/hashicorp/terraform/lang.ReferencesInBlock(0x3067c40, 0xc00056d290, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc00066bc60)
	/home/circleci/project/project/lang/references.go:68 +0x66
github.com/hashicorp/terraform/terraform.(*NodeAbstractResource).References(0xc000460a90, 0x0, 0x0, 0x0)
	/home/circleci/project/project/terraform/node_resource_abstract.go:198 +0x33d
github.com/hashicorp/terraform/terraform.NewReferenceMap(0xc000188b40, 0x1e, 0x1e, 0x1e)
	/home/circleci/project/project/terraform/transform_reference.go:492 +0x24a
github.com/hashicorp/terraform/terraform.(*ReferenceTransformer).Transform(0x4280e00, 0xc00056a000, 0x14, 0x0)
	/home/circleci/project/project/terraform/transform_reference.go:79 +0x71
github.com/hashicorp/terraform/terraform.(*BasicGraphBuilder).Build(0xc00017f8e8, 0x0, 0x0, 0x0, 0x4261160, 0xc00017f918, 0x10e28d0, 0xc0000ca000)
	/home/circleci/project/project/terraform/graph_builder.go:50 +0x477
github.com/hashicorp/terraform/terraform.(*PlanGraphBuilder).Build(0xc000322770, 0x0, 0x0, 0x0, 0x1, 0x2bd4dd8, 0xc000089401, 0xc00017f978)
	/home/circleci/project/project/terraform/graph_builder_plan.go:65 +0xbc
github.com/hashicorp/terraform/terraform.(*Context).Graph(0xc0005eaf00, 0x6, 0xc00017f96e, 0xc000156a00, 0x47, 0x14, 0xbfe9da840bd34090)
	/home/circleci/project/project/terraform/context.go:292 +0x1b8
github.com/hashicorp/terraform/terraform.(*Context).Validate(0xc0005eaf00, 0x0, 0x0, 0x0)
	/home/circleci/project/project/terraform/context.go:675 +0xc9
github.com/hashicorp/terraform/backend/local.(*Local).context(0xc000428f70, 0xc0005b7420, 0x1, 0x1, 0x2, 0xc000303fc0, 0x1, 0x2, 0x0)
	/home/circleci/project/project/backend/local/backend_local.go:120 +0x974
github.com/hashicorp/terraform/backend/local.(*Local).opPlan(0xc000428f70, 0x3067240, 0xc0005da180, 0x3067240, 0xc0005da1c0, 0xc0005b7420, 0xc0005da140)
	/home/circleci/project/project/backend/local/backend_plan.go:70 +0x2a3
github.com/hashicorp/terraform/backend/local.(*Local).Operation.func1(0xc0002b8fb0, 0xc0002b8fc0, 0xc0002b8fd0, 0xc0005b7420, 0xc000428f70, 0xc0005da140, 0xc0002b8fa0, 0x3067240, 0xc0005da180, 0x3067240, ...)
	/home/circleci/project/project/backend/local/backend.go:350 +0x135
created by github.com/hashicorp/terraform/backend/local.(*Local).Operation
	/home/circleci/project/project/backend/local/backend.go:334 +0x36b

```

--- a/examples/linux_general/lld_rule.tf
+++ b/examples/linux_general/lld_rule.tf
@@ -102,6 +102,7 @@ resource "zabbix_trigger_prototype" "free_disk_space_warn" {
   ]
 }
 
+/*
 resource "zabbix_lld_rule_link" "test_link" {
   lld_rule_id = zabbix_lld_rule.filesystem_discovery_rule.id
 
@@ -133,3 +134,4 @@ resource "zabbix_lld_rule_link" "test_link" {
     }
   }
 }
+*/

